minecraft-quilt/minecraft-quilt.json

```diff
@@ -0,0 +1,118 @@
+{
+  "name": "undefined",
+  "display": "Quilt - Minecraft pr",
+  "type": "minecraft-java",
+  "install": [
+    {
+      "type": "javadl",
+      "version": "17"
+    },
+    {
+      "files": [
+        "https://maven.quiltmc.org/repository/release/org/quiltmc/quilt-installer/latest/quilt-installer-latest.jar"
+      ],
+      "type": "download"
+    },
+    {
+      "source": "quilt-installer-*.jar",
+      "target": "quilt-installer.jar",
+      "type": "move"
+    },
+    {
+      "commands": [
+        "java17 -jar quilt-installer.jar install server ${game-version} --download-server --install-dir=."
+      ],
+      "type": "command"
+    },
+    {
+      "target": "server.properties",
+      "text": "server-ip=${ip}\nserver-port=${port}\nmotd=${motd}",
+      "type": "writefile"
+    },
+    {
+      "target": "eula.txt",
+      "text": "eula=${eula}",
+      "type": "writefile"
+    },
+    {
+      "source": "quilt-server-launch.jar",
+      "target": "quilt-server.jar",
+      "type": "move"
+    },
+    {
+      "target": "mods",
+      "type": "mkdir"
+    },
+    {
+      "target": "installer",
+      "type": "mkdir"
+    },
+    {
+      "commands": [
+        "rm quilt-installer.jar"
+      ],
+      "type": "command"
+    }
+  ],
+  "run": {
+    "stop": "stop",
+    "command": "java17 -Xmx${memory}M -Dlog4j2.formatMsgNoLookups=true -jar quilt-server.jar nogui",
+    "workingDirectory": "",
+    "pre": [],
+    "post": [],
+    "environmentVars": {}
+  },
+  "data": {

```

Comment:
Please put the data object after display, per https://github.com/PufferPanel/templates#formattng-rules